onko_dicom.py

```diff
@@ -84,10 +85,19 @@ def open_file(self):
             logger.info("open_file user canceled open operation")
             return
 
-        self.plot_w.plot_dcm(full_path[0])
-        self.close_action.setEnabled(True)
+        try:
+            logger.info("Attempting to graph/open file")
+            self.plot_w.plot_dcm(full_path[0])
+            self.close_action.setEnabled(True)
+            logger.info("successfully opened graph/file")
+
+        except pydicom.errors.InvalidDicomError as err:
+            logger.error("InvalidDicomError, Missing Dicom Header. Error:(%s)", err)
```

Comment:
Will this include the filename?

Reply:
Yep, now includes file names in errors and opening